README.md

```diff
@@ -14,3 +14,14 @@ git clone git@github.com:openfisca/openfisca-ops.git
 ```
 
 Each sub-directory has its own README.md file.
+
+## Services
+
+We use [systemd](https://wiki.debian.org/systemd) services to run the OpenFisca related applications (OpenFisca-France [web](https://api.openfisca.fr/) [APIs](https://api-test.openfisca.fr/), [legislation explorer](https://legislation.openfisca.fr/)).
+
+The service configuration files are located in the `/etc/systemd/system` directory.
```

Comment:
Suggest symlinking in the documentation rather than just replying in the PR comment :wink: